### PR TITLE
Added {} in templates

### PR DESCRIPTION
**Pull request by con-ssc**
**What kind of change does this PR introduce?**
I refactored some template code to harmonize control structures to follow the use of always having {}

**Did you add tests for your changes?**
Tried to use the npm run test but didn't work before changes

**If relevant, link to documentation update:**

**Summary**
Added {} in templates so Microsoft JS guidelines will work on generated manifest.js for bundles like with vue js will pass the quality test.

**Does this PR introduce a breaking change?**
No.

**Other information**

### CODE REVIEW

**Comment by timse** [comment]
to run the tests locally run
`npm link && npm link weback`

maybe we should have that on some post local install step @sokra 
this appears to be a quite common issue :p

**Comment by con-ssc** [comment]
@timse I tried to circumvent that by eslint disables but yes, webpack as devdep makes sense

**Comment by con-ssc** [comment]
@timse @sokra The thing is, there are errors, but there are errors in files I did not touch, so I suspect there where there before.

**Comment by sokra** [comment]
> maybe we should have that on some post local install step @sokra
> this appears to be a quite common issue :p

good idea.

File: test/configCases/plugins/environment-plugin/webpack.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const EnvironmentPlugin = require("../../../../lib/EnvironmentPlugin");
+var EnvironmentPlugin = require("../../../../lib/EnvironmentPlugin");
```
Comment (sokra) [inline test/configCases/plugins/environment-plugin/webpack.config.js]: add `"use strict"` on top instead.

File: test/statsCases/warnings-uglifyjs/webpack.config.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
+//eslint-disable-next-line
```
Comment (sokra) [inline test/statsCases/warnings-uglifyjs/webpack.config.js]: I don't see any reason to disable eslint here. Can't the problem be fixed with code or configuration?

File: test/statsCases/warnings-uglifyjs/webpack.config.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 var webpack = require("webpack");
 
 module.exports = {
 	entry: "./index",
 	output: {
 		filename: "bundle.js"
 	},
+	//eslint-disable-next-line
```
Comment (sokra) [inline test/statsCases/warnings-uglifyjs/webpack.config.js]: same

File: test/configCases/plugins/min-chunk-size/webpack.config.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
-var webpack = require("../../../../")
+//eslint-disable-next-line
+var webpack = require("../../../../");
```
Comment (sokra) [inline test/configCases/plugins/min-chunk-size/webpack.config.js]: same

File: test/configCases/dll-plugin/2-use-dll-without-scope/webpack.config.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
 var path = require("path");
+//eslint-disable-next-line
```
Comment (sokra) [inline test/configCases/dll-plugin/2-use-dll-without-scope/webpack.config.js]: same

File: test/configCases/dll-plugin/1-use-dll/webpack.config.js [hunk 6]
```diff
@@ -1,8 +1,10 @@
+//eslint-disable-next-line
```
Comment (sokra) [inline test/configCases/dll-plugin/1-use-dll/webpack.config.js]: same

File: package.json [hunk 7]
```diff
@@ -50,7 +50,7 @@
     "less": "^2.5.1",
     "less-loader": "^2.0.0",
     "lodash": "^4.17.4",
-    "mocha": "^3.2.0",
+    "mocha": "3.2.0",
```
Comment (sokra) [inline package.json]: why changing this?

**Comment by con-ssc** [comment]
> to run the tests locally run
npm link && npm link weback

> maybe we should have that on some post local install step @sokra
this appears to be a quite common issue :p

For this, webpack needs to be a dep or file in the package.json otherwise it still counts as unpublished in one of the files.

File: test/configCases/plugins/environment-plugin/webpack.config.js (hunk 1 above)
Comment (con-ssc) [inline test/configCases/plugins/environment-plugin/webpack.config.js]: Do you want me to revert it to const and add use strict or just add use strict`?

**Comment by con-ssc** [comment]
Sorry forgot the stats

File: test/configCases/plugins/environment-plugin/webpack.config.js (hunk 1 above)
Comment (sokra) [inline test/configCases/plugins/environment-plugin/webpack.config.js]: const and use strict

**Comment by con-ssc** [comment]
Sorry, I'm not so much used to forking and submitting pull requests.

**Comment by sokra** [comment]
Thanks

**Pull request merged**